ajax/libs/html2pdf.js/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "html2pdf.js",
+  "filename": "html2pdf.bundle.min.js",
+  "description": "Client-side HTML-to-PDF rendering using pure JS",
+  "keywords": [
+    "javascript",
+    "pdf",
+    "html",
+    "client-side",
+    "canvas"
+  ],
+  "author": {
+    "name": "Erik Koopmans",
+    "email": "erik@erik-koopmans.com",
+    "url": "https://www.erik-koopmans.com"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/eKoopmans/html2pdf.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "*.js"
```

Comment:
How about using `!(include|require){,**/*}`, assuming you will release files for a browser under dist, but not under `include` or `require` folder.

Reply:
Sounds right - everything at the root of `dist` is packaged as UMD, for browser and anything else. The `include` and `require` folders are for Node/ES and should be excluded.

Can you point me to a reference for the globbing used in cdnjs? Just want to make sure I understand everything going on in `!(include|require){,**/*}` (and any other options). Thanks!

Reply:
Sure :) 
cdnjs uses [glob](https://www.npmjs.com/package/glob) package, here is reference: https://github.com/cdnjs/autoupdate/blob/master/autoupdate.js#L8. 
`!(include|require){,**/*}` will fetch all subfolder and files under `basePath` except for `include` or `require` folder.